types/ari-client/index.d.ts

```diff
@@ -0,0 +1,2003 @@
+// Type definitions for ari-client 2.2
+// Project: https://github.com/asterisk/node-ari-client
+// Definitions by: Dioris Moreno <https://github.com/dioris-moreno>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+// tslint:disable-next-line:export-just-namespace
+export = Ari;
+export as namespace Ari;
```

Comment:
I don't think this is an UMD module, so this line looks wrong. 

I'm not very familiar with the library. Just double check this is the case and if you feel is alright I can help you with the merge.

Reply:
Asterisk is an open source framework for building communications applications, and "ari-client" is the Node.js library for the Asterisk RESTful Interface (ARI), an asynchronous API that allows developers to build those applications. 

You are right, this is not an UMD library. I rewritten the declaration based on the module-class template (https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html) as follows:

export = Ari;

declare namespace Ari {
    function connect(
        baseUrl: string,
        user: string,
        pass: string,
        callback?: (err: Error, client: Client) => void,
    ): Promise<Client>;

    interface Client extends Resource {
    ...

The idea of exposing types and interfaces in Ari namespace is to allow destructured import of Asterisk objects (Channel, Bridge, Endpoint, etc.). Developers will need to turn on esModuleInterop in order to use an import statement like this:

import Ari, { Channel, Bridge, Endpoint} from 'ari-client';

Reply:
I just pushed the updated version.

Reply:
I don't think the namespace is necessary. I'm inclined to let the tslint rule on and have every interface exported instead. 

If you want to have a syntax like `import Ari, { Channel, Bridge, Endpoint} from 'ari-client';`, all you need to do is enable the `esModuleInterop` as you mentioned.

Reply:
I just pushed the changes you suggested. It is much better.